src/mcs/mcs_types.jl

```diff
@@ -231,6 +231,6 @@ end
 
 function Base.getindex(sim_inst::SimulationInstance, comp_name::Symbol, datum_name::Symbol; model::Int = 1)
     msg = "getindex method for `SimulationInstance` has been replaced with getdataframe function for consistency of return type, please use getdataframe instead"
-    Base.depwarn("$msg, $(stacktrace())", :getindex)
+    error("$msg, $(stacktrace())")
     return sim_inst.results[model][(comp_name, datum_name)]
```

Comment:
If this is an error, then we don't need the `stacktrace()` part (that will come automatically), and we also don't need the `return` statement, because it will never be reached.

Reply:
Good point, was clearly operating with 1/2 my brain there.

Reply:
hmm I think we are still using this stacktrace error message in the other functions we deprecated, should we remove it from all of them now too?